### DISCUSSION TITLE
Error: "Failed to load and execute Flow's environment, Unknown Object Exception" When running flow with Github storage

### DISCUSSION THREAD

**Discussion by samueltober** [PrefectHQ/prefect#4797] [Q&A]
Hi! I am new to Prefect and have been trying to set up a flow with Github storage. I have the following flow file in my repo: 

    `
    @task
    def say_hello():
        logger = prefect.context.get('logger')
        logger.info("Hello world.")
    
    
    with Flow("test-flow") as flow:
        flow.storage = GitHub(  repo="voiapp/Swap Task Forecast", 
                                access_token_secret=PrefectSecret("github-token"),
                                path="/test/prefect-test.py")
        say_hello()
    
    flow.register(project_name="Swap Task Forecast")
    `

The flow registers in the UI, however, when i run it with, I get the following error:

"Failed to load and execute Flow's environment, Unknown Object Exception"

I cannot seem to find the issue, any help would be much appreciated!

**Comment by zanieb** [comment]
I believe `Unknown Object Exception` is a 404 from Github, it cannot find the repo/path you've provided with the token. Usually there should be some more information attached to this error though. I think the issue here may be that `access_token_secret` should be the name of the secret to lookup (e.g. `"github-token"`) rather than a secret object.
